src/test/java/teammates/test/cases/webapi/CreateAccountActionTest.java

```diff
@@ -38,39 +39,16 @@ protected void testExecute() throws Exception {
         String email = "jamesbond89@gmail.tmt";
         String institute = "TEAMMATES Test Institute 1";
 
-        ______TS("Not enough parameters");
```

Comment:
Why remove these test cases? You can minimally test for the same thing, just converting parameters to body (i.e. no body sent, partial body etc.)

Reply:
Jeremy is correct. We shall modify the tests instead of removing them.